frontend/public/components/overview/project-overview.tsx

```diff
@@ -237,6 +240,7 @@ const ProjectOverviewListItem = connect<ProjectOverviewListItemPropsFromState, P
           {name}
         </button>
         {current && <React.Fragment>,&nbsp;<ControllerLink controller={current} /></React.Fragment>}
+        {isDeleting && <ResourceItemDeleting />}
```

Comment:
This is a little more robust than `_.has` since `_.has` will return true for null/empty string. You might add `deletionTimestamp` to the destructuring on line 214.

```suggestion
        {obj.metadata.deletionTimestamp && <ResourceItemDeleting />}
```